mungegithub/mungers/submit-queue.go

```diff
@@ -987,6 +987,8 @@ const (
 	ghE2EFailed             = "Second github e2e run failed."
 	unmergeableMilestone    = "Milestone is for a future release and cannot be merged"
 	headCommitChanged       = "This PR has changed since we ran the tests"
+	ghReviewStateUnclear    = "Cannot get gh reviews status"
+	ghReview                = "This pr has no gh approve or has changes request"
```

Comment:
s/gh approve/GH 'approved' review/  This should be carefully worded to distinguish the `approved` label from github's concept of approved reviews.